src/v/test_utils/runfiles.cc

```diff
@@ -0,0 +1,29 @@
+#include "test_utils/runfiles.h"
+
+#ifdef BAZEL_TEST
+#include "tools/cpp/runfiles/runfiles.h"
+#endif
+
+#include <fmt/format.h>
+
+#include <memory>
+
+namespace test_utils {
+
+std::optional<std::string>
+get_runfile_path([[maybe_unused]] std::string_view path) {
+#ifdef BAZEL_TEST
+    using bazel::tools::cpp::runfiles::Runfiles;
+    std::string error;
+    std::unique_ptr<Runfiles> runfiles(
+      Runfiles::CreateForTest(BAZEL_CURRENT_REPOSITORY, &error));
+    if (runfiles == nullptr) {
+        throw std::runtime_error(error);
+    }
+    return runfiles->Rlocation(fmt::format("_main/{}", path));
```

Comment:
Just curious what _main is here? Looked through https://github.com/bazelbuild/bazel/blob/master/tools/cpp/runfiles/runfiles_src.h but it wasn't all that helpful

Reply:
apparently it's the name of the root of the project. i think it only matters when building modules or referencing data inside of other modules. not entirely sure what the precise meaning is.